source/_integrations/spc.markdown

```diff
@@ -34,12 +34,25 @@ api_url:
   required: true
   type: string
 ws_url:
-  description: URL of the SPC Web Gateway websocket, e.g., `ws://<ip>:8088`.
+  description: URL of the SPC Web Gateway websocket, e.g., `ws://<ip>:8088/ws/spc`.
   required: true
   type: string
 {% endconfiguration %}
 
-Supported sensors will be automatically discovered and added, however they will be hidden by default.
+Supported sensors will be automatically discovered and added, however they will be hidden by default, and unless unhidden they will not produce any history or work in automations.
```

Comment:
Let's remove this part altogether.
With Lovelace, there is no reason to hide these sensors.

home-assistant/home-assistant#30699

Reply:
The reason for the change was that the integration doesn't work unless the sensors are unhidden. The integration sets them to hidden, and just changing them in Lovelace reverts back to hidden at restart. At the very least we need to keep the information about the sensors not working unless they are manually unhidden. 

This was the only way I could make it work consistently.

Reply:
@linkztream If you've clicked the link I've put in my last message, you would have found I have changed the integration in Home Assistant to not hide those sensors anymore.

Please remove the customization additions, as they are not needed anymore.

Reply:
You're absolutely right, my bad. Sorry for not reading properly! This was my first pull request ever, so got a little bit lost in it all.

With that change to the integration (and some better reading comprehension on my part) I completely agree with you.